tests/Aspire.Hosting.Tests/ProjectResourceTests.cs

```diff
@@ -177,11 +177,34 @@ public void ExcludeLaunchProfileAddsAnnotationToProject()
     }
 
     [Fact]
-    public async Task VerifyManifest()
+    public void DisabledForwadedHeadersAddsAnnotationToProject()
     {
         var appBuilder = CreateBuilder();
 
-        appBuilder.AddProject<TestProjectWithLaunchSettings>("projectName");
+        appBuilder.AddProject<Projects.ServiceA>("projectName").DisableForwadedHeadersOnPublish();
+        using var app = appBuilder.Build();
+
+        var appModel = app.Services.GetRequiredService<DistributedApplicationModel>();
+
+        var projectResources = appModel.GetProjectResources();
+
+        var resource = Assert.Single(projectResources);
+        // DisableForwardedHeadersAnnotation isn't public, so we just check the type name
```

Comment:
Feels bad

Reply:
I just copied the test that was already there...

Reply:
If you want I can make it IVT and fix both tests

Reply:
Fixed